server/src/main/java/com/defold/extender/services/data/DefoldSdk.java

```diff
@@ -0,0 +1,52 @@
+package com.defold.extender.services.data;
+
+import java.io.File;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+import com.defold.extender.services.DefoldSdkService;
+
+public class DefoldSdk implements AutoCloseable {
```

Comment:
Class to wrapping information about defold sdk. Implement as auto closeable resource.